recipes/wasmtime/all/conanfile.py

```diff
@@ -65,26 +60,33 @@ def validate(self):
             self.output.warn(msg)
 
         try:
-            self.conan_data["sources"][self.version][str(self.settings.os)]
+            self.conan_data["sources"][self.version][self._sources_key][str(self.settings.arch)]
         except KeyError:
             raise ConanInvalidConfiguration("Binaries for this combination of architecture/version/os not available")
 
+        if (self.settings.compiler, self.settings.os) == ("gcc", "Windows") and self.options.shared:
+            # FIXME: https://github.com/bytecodealliance/wasmtime/issues/3168
```

Comment:
This was merged in

Reply:
Indeed, but we will need to wait for the next release to have shared mingw avaialble.

Reply:
Ahh I did not look, but thats for checking!